README.md

```diff
@@ -98,23 +98,40 @@ The following steps are for Linux and OSX.
     # Setup the workspace
     mkdir -p ~/ws/src
     cd ~/ws/src
+
     # Download needed software
     git clone https://github.com/osrf/ros_ign.git -b ros2
     ```
 
-1. Install dependencies (this may also install Ignition):
+1. Install ROS dependencies:
 
     ```
     cd ~/ws
-    rosdep install --from-paths src -i -y --rosdistro foxy
+    rosdep install --from-paths src -i -y --rosdistro foxy \
```

Comment:
On Focal, rosdep should be able to install Citadel dependencies, because that's the official version for Foxy and Noetic.

After Edifice is released, that's also going to be made available through rosdep for Focal, because that will be the official version for Galactic.

Summarizing:

Ignition | Ubuntu | rosdep-installable
-- | -- | --
Citadel | Bionic | no
Citadel | Focal | yes
Dome | Bionic | no
Dome | Focal | no
Edifice | Bionic | no
Edifice | Focal | not yet

So how about, instead of skipping all keys, using `-r` to keep installing despite errors, and tell users to install Ignition from packages.osrfoundation.org if they fail through `rosdep`?

Reply:
Good to know about Ignition `rosdep` support! Updated with `-r` (https://github.com/ignitionrobotics/ros_ign/pull/142/commits/ac729704271021d2fc309f75938cb901b76ce96c).

I wasn't really sure where to point users if `rosdep` installation fails without making it sound confusing. Especially since error(s) will be returned even if Ignition workspace overlay is sourced prior to calling `rosdep install`.